src/Makefile

```diff
@@ -56,6 +56,7 @@ CUBEAI_DIR=stm32cubeai
 CMSIS_DIR=hal/cmsis
 MICROPY_DIR=micropython
 GENX320_DIR=drivers/genx320
+BOSON_DIR=drivers/boson
```

Comment:
You should also export this in `omv/common/export.mk`

Reply:
Done